Makefile

```diff
@@ -53,3 +54,6 @@ install:
 uninstall:
 	/bin/rm -f $(DESTDIR)/$(MODULE_DESTDIR)/nv_peer_mem.ko
 	if [ ! -n "$(DESTDIR)" ]; then $(DEPMOD) -r -ae $(KVER);fi;
+
+gen_nv_symvers:
+	$(PWD)/create_nv.symvers.sh $(KVER)
```

Comment:
Okay.
OT: Debian/rules should build the dkms package, not the module, right? Why does it care about the kernel version? 
By the way, you should also remove nv.symvers from the dkms package.